api/solarperformanceinsight_api/routers/parameters.py

```diff
@@ -17,6 +17,25 @@
     .astype(float)
 )
 
+# Database does not provide the fields 'cells_in_series', 'EgRef', 'dEgdT'
+# which have defaults in the model.
+cec_keys = [
+    "alpha_sc",
+    "a_ref",
+    "I_L_ref",
+    "I_o_ref",
+    "R_sh_ref",
+    "R_s",
+    "Adjust",
+    "N_s",
+]
+cec_module_params = (
+    retrieve_sam("CECMod")
+    .loc[cec_keys]
+    .astype(float)
```

Comment:
I guess json/javascript only have "number" and don't differentiate between int and float, right? just because now cells in series is an int and should roundtrip back as the same int

Reply:
Yes, javascript only has the `number` type, but JSON can represent integers correctly. The model is deserializing the `cells_in_series` field to the correct int before transmitting it.
On the dashboard, this is being handled by checking the schema for the `integer` type and setting the step of the input to 1, so that an integer is included in the JSON when posting.